modules/apps/translation/translation-translator-aws/src/main/java/com/liferay/translation/translator/aws/internal/translator/AWSTranslator.java

```diff
@@ -119,7 +119,13 @@ public String getTargetLanguageId() {
 	private String _getLanguageCode(String languageId) {
 		List<String> list = StringUtil.split(languageId, CharPool.UNDERLINE);
 
-		return list.get(0);
+		String languageCode = list.get(0);
+
+		if (languageCode.equals("in")) {
+			return "id";
+		}
+
+		return languageCode;
```

Comment:
I think we need to add a comment to avoid confusion. Something like:
```
// See LPS-23561. Azure expects ISO-639 locale names; "in" was renamed to "id" in ISO-639:1989.

...
```

Reply:
Done (I changed it a little bit to adjust it to other formatting patterns I've seen).